xbmc/cores/AudioEngine/Sinks/AESinkAUDIOTRACK.cpp

```diff
@@ -738,29 +709,50 @@ unsigned int CAESinkAUDIOTRACK::AddPackets(uint8_t **data, unsigned int frames,
       loop_written = 0;
     }
   }
-  unsigned int written_frames = (unsigned int) (written/m_format.m_frameSize);
+  unsigned int written_frames = (unsigned int)(written / m_format.m_frameSize);
```

Comment:
Can you replace the C style cast with a C++ style cast as you already touched the line?

Reply:
Yes - will do. Thanks.